### ISSUE TITLE
Missing sources for DTB files

### ISSUE THREAD

**Issue by proski** [dhruvvyas90/qemu-rpi-kernel#106]
The DTB files are missing sources (the DTS files). Sure, I can decompile them with `dtc`, but I'm missing the comments. It's likely that the sources had copyright statements in the comments. Stripping copyright is really bad! I don't know how to contact the file author. I cannot post a PR modifying the device tree.

If the DTB files were produced as a result of Link kernel builds, please specify it so that others can reproduce those builds.

**Comment by manoj153** [comment]
Yes I'm hoping for a DTS too. i don't feel right to use a random compiled DTB or atleast point to the canonical location of the dts?
